src/main/java/org/prebid/server/handler/VersionHandler.java

```diff
@@ -37,17 +37,19 @@ public static VersionHandler create(String revisionFilePath, JacksonMapper mappe
             revision = mapper.mapper().readValue(ResourceUtil.readFromClasspath(revisionFilePath), Revision.class);
         } catch (IllegalArgumentException | IOException e) {
             logger.warn("Was not able to read revision file {0}. Reason: {1}", revisionFilePath, e.getMessage());
-            revision = Revision.of(DEFAULT_REVISION_VALUE);
+            revision = Revision.of(NOT_SET, NOT_SET);
         }
-        return new VersionHandler(revision.commitHash == null ? Revision.of(DEFAULT_REVISION_VALUE) : revision, mapper);
+        return new VersionHandler(revision, mapper);
     }
 
     /**
      * Responds with commit revision.
      */
     @Override
     public void handle(RoutingContext context) {
-        final RevisionResponse revisionResponse = RevisionResponse.of(revision.commitHash);
+        final RevisionResponse revisionResponse = RevisionResponse.of(
```

Comment:
Can we compose full version info in `create()` method, bc no reason to recreate the same state?